Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -200,6 +207,13 @@ NetworkStorageManager::NetworkStorageManager(PAL::SessionID sessionID, IPC::Conn
             auto saltPath = FileSystem::pathByAppendingComponent(m_path, "salt"_s);
             m_salt = valueOrDefault(FileSystem::readOrMakeSalt(saltPath));
         }
+#if PLATFORM(IOS_FAMILY)
+        // Exclude LocalStorage directory to reduce backup traffic. See https://webkit.org/b/168388.
+        if (m_shouldUseCustomPaths && !m_customLocalStoragePath.isEmpty()) {
+            FileSystem::makeAllDirectories(m_customLocalStoragePath);
+            FileSystem::setExcludedFromBackup(m_customLocalStoragePath, true);
+        }
+#endif
```

Comment:
Now that we exclude all storage from backup by default, should we start _including_ LocalStorage when we do back up? It seems affordable to do that, and possibly more in line with what a user would want for a frequently visited site.

Reply:
This is for the case where clients use old storage layout; we still exclude LocalStorage to keep the old behavior.